cmd/goal/node.go

```diff
@@ -458,6 +459,34 @@ func makeStatusString(stat model.NodeStatusResponse) string {
 		if stat.StoppedAtUnsupportedRound {
 			statusString = statusString + "\n" + fmt.Sprintf(catchupStoppedOnUnsupported, stat.LastRound)
 		}
+
+		if stat.UpgradePropose != nil {
```

Comment:
Same as below, no need to display this once the vote is complete.
```suggestion
		if stat.UpgradePropose != nil && stat.LastVersion != stat.NextVersion {
```

Reply:
done